languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,35 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="ACCORD_TOUT_ADJECTIF" name="accord tout adjectif" default="temp_off">
+          <rule>
+              <pattern>
+                  <token postag="D f s.*" postag_regexp="yes"></token>
```

Comment:
Regular expression like this can be used here to include contractions (aux, du... ):
`<token postag="(P\+)?D .*" postag_regexp="yes"/>`